### PR TITLE
Storyboard for fast user switching

### PR DESCRIPTION
**Pull request by jakob223**
Desktop version isn't in stories and instead is with mock data under a feature flag, since it seemed hard to do justice to it storybook-wise given that it is in a floating menu.

Layout changes on desktop w/r/t dividers as implemented here were discussed with design.
Mobile version has slightly shorter dividers than in the design because that's how `ListItem2` is implemented. @cecileboucheron is that okay?

Confused as to why the `rightActions` are showing up over to the left. I plan to add one more commit to make that red once I figure out how to make it come up on the right.
 - [ ] fix right action
 - [ ] merge conflict

image

CC @keybase/y2ksquad

### CODE REVIEW

**Comment by cecileboucheron** [comment]
@jakob223 @adamjspooner 
Why didn't you use NameWithIcon like on desktop?

**Comment by cecileboucheron** [comment]
The Sign Out should be on the upper right corner, and red:
image

**Comment by jakob223** [comment]
@cecileboucheron I used `ListItem2` because it provided letting me click on any of the parts of the row, the right sizes for the icons, and dividers for free. If you prefer the dividers to be full width, I can figure out how to do that with either listitem or namewithicon.

w/r/t the Sign Out button, I know - that's the "right action" I mentioned being on the left side.

**Comment by jakob223** [comment]
Talked with cécile, will switch dividers on mobile to match desktop.

Bump @keybase/react-hackers !

**Comment by jakob223** [comment]
Bump @keybase/react-hackers !

File: shared/router-v2/account-switcher/index.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    onClick={() => {
+      onSelectAccount(entry.username)
+    }}
+    containerStyle={styles.row}
+    avatarStyle={!entry.signedIn && styles.avatarSignedOut}
+  />
+)
+
+const AccountSwitcherMobile = (props: Props) => (
+  <Kb.ScrollView>
```
Comment (buoyad) [inline shared/router-v2/account-switcher/index.tsx]: `alwaysBounceVertical={false}`

**Comment by jakob223** [comment]
Made the requested changes in the Y2K-127 branch. Since that's being kept up to date and this stuff isn't useful until that is merged anyway, I'm going to close this PR and merge this stuff in with that stuff.

**Pull request closed without merging**